dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/DefaultTrackerImportService.java

```diff
@@ -186,9 +191,10 @@ protected void commitBundle( TrackerImportParams params, TrackerImportReport imp
 
         TrackerBundleReport bundleReport = trackerBundleService.commit( trackerBundle );
```

Comment:
Is this method ever returning a null bundleReport?

Reply:
nope, but in case of TrackerBundleMode.VALIDATE it will return an instance where there's no ENROLLMENT nor EVENT keys in typeReportMap, which is why it was going NPE. To make it safer I checked for null starting from bundleReport which is redundant, yet not hitting performances at all, so it should be fine